client/src/app/site/motions/services/motion-import.service.ts

```diff
@@ -264,12 +264,12 @@ export class MotionImportService extends BaseImportService<ViewMotion> {
             return null;
         }
         blockString = blockString.trim();
-        let existingBlock = this.motionBlockRepo.getMotionBlockByTitle(blockString);
-        if (!existingBlock) {
-            existingBlock = this.motionBlockRepo.getMotionBlockByTitle(this.translate.instant(blockString));
+        let existingBlock = this.motionBlockRepo.getMotionBlocksByTitle(blockString);
+        if (!existingBlock.length) {
+            existingBlock = this.motionBlockRepo.getMotionBlocksByTitle(this.translate.instant(blockString));
         }
-        if (existingBlock) {
-            return { id: existingBlock.id, name: existingBlock.title };
+        if (existingBlock.length) {
+            return { id: existingBlock[0].id, name: existingBlock[0].title };
         } else {
```

Comment:
reflecting changes in motion block repo. We still use the first one coming, though, as we don't really expect more than one